.pre-commit-config.yaml

```diff
@@ -13,6 +13,7 @@ repos:
         hooks:
               - id: black
                 files: python/cuspatial/.*
+                args: ["--config", "pyproject.toml"]
```

Comment:
I see you're using black as a pre-commit hook. How do you enable this? Is it on by default and I haven't noticed?

Reply:
You can either run `pre-commit` to manually run it prior committing or let git run it automatically!

https://pre-commit.com/#3-install-the-git-hook-scripts

Reply:
The change here is to make black's config file available to all subpackages in the repo. Unlike cudf (before https://github.com/rapidsai/cudf/pull/11258), `cuspatial-python/pyproject.toml` doesn't define black configs, so when you try to run `python -m black` in the subpackage, it may disregard the config file at root level. This is a minor bug fix unrelated to the refactor.